### PR TITLE
ARTEMIS-519  protocol-manager-factory is not implemented on ConnectionFactoryControl

### PR DESCRIPTION
**Pull request by bayern39**
https://issues.apache.org/jira/browse/ARTEMIS-519

### CODE REVIEW

**Comment by mtaylor** [comment]
@bayern39 Hi, you have several check style issues in your commit.  Could you please fix these.  To find these locally build with the dev profile

`mvn compile -Pdev`

**Comment by bayern39** [comment]
@mtaylor  OK,i did it ,thanks for your tip!

File: artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/management/impl/JMSConnectionFactoryControlImpl.java [hunk 1]
```diff
@@ -432,6 +432,16 @@ public int getTransactionBatchSize() {
    }
 
    @Override
+   public void setProtocolManagementFactoryStr(String protocolManagerFactoryStr) {
+      cfConfig.setProtocolManagerFactoryStr(protocolManagerFactoryStr);
+      recreateCF();
+   }
+
+   @Override
+   public String geProtocoltManagementFactoryStr() {
```
Comment (jbertram) [inline artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/management/impl/JMSConnectionFactoryControlImpl.java]: "Protocol" is misspelled in this method name.

**Comment by bayern39** [comment]
@jbertram  i have fixed it,thanks!

**Pull request closed without merging**